<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_window</class>
 <widget class="QDialog" name="search_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>481</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>9</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 243, 243);
selection-background-color: rgb(83, 83, 83);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_bt">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Size of array</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="size_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 243, 243);
selection-background-color: rgb(83, 83, 83);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enter_ip">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QComboBox" name="speed_comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>55</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 243, 243);
selection-background-color: rgb(83, 83, 83);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Speed of Animation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="play_tab">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Number to Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="no_lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 243, 243);
selection-background-color: rgb(83, 83, 83);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enter_no">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Number of Comparisions</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
color:white;</string>
    </property>
    <property name="text">
     <string>Select Algorithm</string>
    </property>
   </widget>
   <widget class="QComboBox" name="algo_comboBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 243, 243);
selection-background-color: rgb(83, 83, 83);</string>
    </property>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="info_edit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>111</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(188, 188, 188);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
